system/web/context/EventHandlerBean.cfc

```diff
@@ -61,57 +66,62 @@ component accessors="true"{
 		variables.viewDispatch		= false;
 		variables.actionMetadata 	= {};
 		variables.handlerMetadata 	= {};
-		
+		variables.metadataLoaded 	= false;
+
 		return this;
 	}
 
 	/************************************** PUBLIC RETURN BACK SETTERS *********************************************/
-	
-	function setIsPrivate( required isPrivate ){ 
+
+	function setIsPrivate( required isPrivate ){
 		variables.isPrivate = arguments.isPrivate;
-		return this; 
+		return this;
 	}
-	function setHandler( required handler ){ 
+	function setHandler( required handler ){
 		variables.handler = arguments.handler;
-		return this; 
+		return this;
 	}
-	function setMethod( required method ){ 
+	function setMethod( required method ){
 		variables.method = arguments.method;
-		return this; 
+		return this;
 	}
-	function setModule( required module ){ 
+	function setModule( required module ){
 		variables.module = arguments.module;
-		return this; 
+		return this;
 	}
-	function setMissingAction( required missingAction ){ 
+	function setMissingAction( required missingAction ){
 		variables.missingAction = arguments.missingAction;
-		return this; 
+		return this;
 	}
-	function setViewDispatch( required viewDispatch ){ 
+	function setViewDispatch( required viewDispatch ){
 		variables.viewDispatch = arguments.viewDispatch;
-		return this; 
+		return this;
 	}
-	function setInvocationPath( required invocationPath ){ 
+	function setInvocationPath( required invocationPath ){
 		variables.invocationPath = arguments.invocationPath;
-		return this; 
+		return this;
 	}
-	function setActionMetadata( required actionMetadata ){ 
+	function setActionMetadata( required actionMetadata ){
 		variables.actionMetadata = arguments.actionMetadata;
-		return this; 
+		return this;
 	}
-	function setHandlerMetadata( required handlerMetadata ){ 
+	function setHandlerMetadata( required handlerMetadata ){
 		variables.handlerMetadata = arguments.handlerMetadata;
-		return this; 
+		return this;
+	}
+	function setMetadataLoaded( required metadataLoaded ){
```

Comment:
why add this setter if accessors is already on?

Reply:
Just in line with all the others you have is all. to make chainable. Not necessary.